test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -690,15 +717,13 @@ def test_successful_capture_with_mdd_fields
     assert_successful_response(capture)
   end
 
+  # this test should probably be removed, the fields do not appear to be part of the
+  # most current XSD file, also they are not added to the request correctly as top level fields
   def test_merchant_description
     merchant_options = {
-      merchantInformation: {
-        merchantDescriptor: {
-          name: 'Test Name',
-          address1: '123 Main Dr',
-          locality: 'Durham'
-        }
-      }
+      merchant_descriptor_name: 'Test Name',
+      merchant_descriptor_address1: '123 Main Dr',
+      merchant_descriptor_locality: 'Durham'
     }
```

Comment:
This tests passes as implemented, but that's because the fields are not added as top level options. I was on the fence about whether to deal with this issue now or leave it for a later clean up task, so I'd love to hear your thoughts.

Reply:
This test failure just exposes that this field implementation was already broken